tests/data/classic_windfarm.py

```diff
@@ -0,0 +1,302 @@
+from cognite.client.data_classes import (
```

Comment:
As you have put some thoughts into this classic dataset I am wandering do we want to maintain `./tests/data/class_cdf_extractore_data`, which I created with a similar idea (when testing classic cdf extractors) but I stored instances/resources as YAML files ?